src/DependencyInjection/ConnecthollandUserExtension.php

```diff
@@ -44,21 +44,32 @@ public function prepend(ContainerBuilder $container)
         $config = $this->getResourceOwnersConfiguration($container);
         $container->prependExtensionConfig('hwi_oauth', $config);
 
-        if ($container->hasExtension('lexik_jwt_authentication')) {
-            $config = $this->getJwtConfiguration($container);
-            $container->prependExtensionConfig('lexik_jwt_authentication', $config);
-        }
+        $this->prependJwtConfiguration($container);
+        $this->prependApiPlatformConfiguration($container);
     }
 
-    private function getJwtConfiguration(ContainerBuilder $container): array
+    private function prependApiPlatformConfiguration(ContainerBuilder $container): void
     {
-        $config = [
-            'secret_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_SECRET_KEY)'), true),
-            'public_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PUBLIC_KEY)'), true),
-            'pass_phrase' => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PASSPHRASE)'), false),
-        ];
+        if ($container->hasExtension('api_platform')) {
+            $config = [
+                'mapping'  => [
+                    'paths' => [sprintf('%s/../Message', __DIR__)],
```

Comment:
Can't you get the class paths here of the classes you'd like to add, in that way renaming and/or refactoring isn't a pain.

```
'paths' => [dirname((new \ReflectionClass(Authenticate::class))->getFileName())],
```